subprojects/plugins/src/integTest/groovy/org/gradle/groovy/compile/BasicGroovyCompilerIntegrationSpec.groovy

```diff
@@ -77,12 +77,47 @@ abstract class BasicGroovyCompilerIntegrationSpec extends MultiVersionIntegratio
         groovyClassFile('Groovy.class').exists()
         groovyGeneratedSourceFile('Groovy$$Generated.java').exists()
         groovyClassFile('Groovy$$Generated.class').exists()
+        outputDoesNotContain('Annotation processing is not supported when incremental Groovy compilation is enabled.')
```

Comment:
This check also needs updating. I think it can be removed, since the test fixture checks automatically that we are not getting a deprecation warning.